server/prisma/datamodel.prisma

```diff
@@ -0,0 +1,67 @@
+enum TaskStatus {
+  COMPLETED
+  UNCOMPLETED
+}
+
+enum UserRole {
+  NEWBIE
+  BUDDY
+}
+
+type NewbieTask {
+  id: ID! @id
+  createdAt: DateTime! @createdAt
+  description: String! @default(value: "")
+  title: String! @default(value: "")
+  status: TaskStatus! @default(value: UNCOMPLETED)
+  newbie: Newbie!
+  notes: String
+}
+
+type BuddyTask {
+  id: ID! @id
+  createdAt: DateTime! @createdAt
+  description: String! @default(value: "")
+  title: String! @default(value: "")
+  status: TaskStatus! @default(value: UNCOMPLETED)
+  newbie: Newbie!
+}
+
+type Buddy {
+  id: ID! @id
+  name: String!
+  email: String! @unique
+  role: UserRole! @default(value: BUDDY)
+  password: String!
+  position: String
+  photo: String
+  phoneNumber: String
+  startDate: DateTime
+  notifications: Boolean! @default(value: false)
```

Comment:
Which functionality is this one referring to? The name is not self explanatory for me.

Reply:
@duiliopastorelli Ok, so here's the `notifications` on the screenshots.  
Any suggestions how can we describe it better: `systemNotifications`, `pushNotifications`, `emailNotifications`?

Reply:
`allowPushedNotifications` - false by default